.github/workflows/main.yaml

```
@@ -4,38 +4,49 @@
+        uses: docker/login-action@v2
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Run build 
-        run: make docker-build
-
-      - name: Push Docker images
+      - name: Push Docker Images
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:3:41: Double quote to prevent globbing and word splitting

(shellcheck)